chain/gen/genesis/genesis.go

```diff
@@ -406,8 +406,8 @@ func VerifyPreSealedData(ctx context.Context, cs *store.ChainStore, stateroot ci
 	verifNeeds := make(map[address.Address]abi.PaddedPieceSize)
 	var sum abi.PaddedPieceSize
 
-	nwv := func(context.Context, abi.ChainEpoch) runtime.NetworkVersion {
-		return runtime.NetworkVersion1
+	nwv := func(context.Context, abi.ChainEpoch) network.Version {
+		return network.Version1
```

Comment:
Not Version2?

Also, I feel like this belongs in `build/`, `NewNetworkVersion` or something like that

Reply:
No, we're still version 1 (from the PoV of actors, which is what cares about this).

Reply:
`NewestNetworkVersion`?

Reply:
Yeeee, fixed (and also in genesis miners setup)